lib/Version.php

```diff
@@ -24,6 +24,8 @@ final class Version
 {
     const PACKAGE_NAME = 'pimcore/pimcore';
 
+    const MAJOR_VERSION = 10;
```

Comment:
This isn't used anywhere?

Reply:
this is added to support Pimcore bundles on both versions 10 & 11 with BC layer, which checks the major version from this class and implement version specific code.

Reply:
e.g. https://github.com/pimcore/customer-data-framework/blob/d0d20f591d2f8625e3a6732b33bbcf041c10abbd/src/Model/ActivityList/ActivityListInterface.php#L23